app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Good job with the associations, but because according to the ERD diagram, it is called author and not user, then this association will look like below. Please kindly implement this.

`belongs_to :author, class_name: 'User'`